lib/irc/ClientPool.js

```diff
@@ -204,6 +204,20 @@ ClientPool.prototype._getNumberOfConnections = function(server) {
     return numConnectedNicks;
 };
 
+ClientPool.prototype.countTotalConnections = function() {
+    var count = 0;
+
+    Object.keys(this._virtualClients).forEach((domain) => {
```

Comment:
I'd prefer it if you used `_getNumberOfConnections` instead, given you are reproducing the same code and they are arguably doing the same thing (albeit one has a `server` and the other does not).

You can get servers from a domain via https://github.com/matrix-org/matrix-appservice-irc/blob/0f82ec6599546473672ea84d2fe439207b0eba9a/lib/bridge/IrcBridge.js#L648